data_preparation.py

```diff
@@ -0,0 +1,103 @@
+import tensorflow as tf
+from tensorflow.keras.preprocessing.image import ImageDataGenerator
+from sklearn.model_selection import train_test_split
+import keras.models
+import numpy as np
+import os
+from PIL import Image
+import matplotlib.pyplot as plt
+
+#Get the directories of burst and non-burst images
+def get_imgs_directory():
+    c_dir = os.getcwd()
+    relative_burst_dir = os.path.join('data', 'burst')
+    relative_nburst_dir = os.path.join('data', 'no_burst')
+
+    burst_image_dir = os.path.join(c_dir, relative_burst_dir)
+    nburst_image_dir = os.path.join(c_dir, relative_nburst_dir)
+    
+
+    print(burst_image_dir)
+    print(nburst_image_dir)
+    
+    return burst_image_dir, nburst_image_dir
+
+#resize and fill the images to be the target size: likely 256x256 for possible use in CNN's
+def resize_and_fill(image_path, target_size):
```

Comment:
I would rather see some keras functions here but if it works, that is OK. However, it's not very controllable. I would rather see keras functions, such as https://www.tensorflow.org/api_docs/python/tf/keras/layers/Resizing

Reply:
I think using Keras has higher chance of leading to distortions